setup.py

```diff
@@ -17,7 +17,8 @@
           'requests>=2.18.4',
           'future',
           'nbsphinx',
-          'jupyter_client'
+          'jupyter_client',
```

Comment:
what's this for?

Reply:
I removed it, I was trying to use a fancy progress bar.